pkg/internal/codegen/parse/util.go

```diff
@@ -375,7 +375,7 @@ func parseDescription(res []string) string {
 	var temp strings.Builder
 	var desc string
 	for _, comment := range res {
-		if !(strings.Contains(comment, "+kubebuilder") || strings.Contains(comment, "+optional")) {
+		if !(strings.Contains(comment, "+kubebuilder") || strings.Contains(comment, "+optional") || strings.Contains(comment, "+required")) {
```

Comment:
@sttts we'll also need to add an exception for `+nullable` or take a better approach than OR'ing every tag here. Maybe we could assume if a line begins with `+` it's a tag and not to include it in the description?